examples/iterate-over-table-rows/README.md

```diff
@@ -0,0 +1,17 @@
+# Iterate Over Table Rows
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to iterate over table rows.
```

Comment:
Why iterate 'over'?

Reply:
@dirk-pieterse `iterate over` is a much more common way to phrase this, especially when it comes to programming concepts. There is some info on it [here](https://stackoverflow.com/questions/2715388/is-it-iterate-through-or-iterate-over-something).

Reply:
Ok. I'm familiar with programming concepts, such as 'to iterate something', but I've not come across 'iterate over' before.